src/components/MealWrapper/MealWrapper.jsx

```diff
@@ -1,13 +1,68 @@
-import styles from "./MealWrapper.module.scss";
+import styles from './MealWrapper.module.scss'
+import foodPhoto from '../../images/foodPhoto.jpg'
 function MealWrapper() {
-  return (
-    <div className={styles.MealWrapper}>
-      MealWrapper1
-      <div className={styles.MealCard1}>Card 1</div>
-      <div className={styles.MealCard2}>Card 2</div>
-      <div className={styles.MealCard3}>Card 3</div>
-    </div>
-  );
+    return (
+        <section className={styles.MealWrapper}>
+            <h2 className={styles.MealwrapparHeader}>FAVORITER</h2>
+            <div className={styles.MealCard}>
+                <div className={styles.MealCardPicContainer}>
+                    <img
+                        className={styles.MealCardPic}
+                        src={foodPhoto}
+                        alt="pic"
```

Comment:
This is not a great alt text :) Consider something more descriptive or leave empty, if irrelevant. I.e. `<img alt="">`